src/events/message/messageCreate.ts

```diff
@@ -83,7 +83,7 @@ export default class extends Event {
 
 		try {
 			await command.run(context, options.args);
-		} catch (err) {
+		} catch (err: any) {
```

Comment:
An alternative solution to this issue would be to make logError accept the type `unknown` in addition to `Error` for the first argument

Reply:
Worth changing in the future